crates/orchestrator/src/cli/mod.rs

```diff
@@ -443,54 +443,51 @@ pub mod validate_params {
         ethereum_args: &EthereumSettlementCliArgs,
         starknet_args: &StarknetSettlementCliArgs,
     ) -> Result<SettlementValidatedArgs, String> {
-        if ethereum_args.settle_on_ethereum {
-            if starknet_args.settle_on_starknet {
-                return Err("Cannot settle on both Ethereum and Starknet".to_string());
+        match (ethereum_args.settle_on_ethereum, starknet_args.settle_on_starknet) {
+            (true, true) => Err("Cannot settle on both Ethereum and Starknet".to_string()),
```

Comment:
i think it was match before but i recommended changing because it was cleaner but if you also find this better let's do this